src/api/DataSource/types.ts

```diff
@@ -78,6 +78,7 @@ export interface DataSourceUserInfo {
   features: DataSourceFeatures;
   defaultStyles?: IDataSourceDefaultStyles; // defined if withStyles or withCaptions was set to true in the request and the data-source is connected
   defaultCaptions?: ICaptions;
+  defaultPropertiesOrder?: IPropertiesOrder;
```

Comment:
when is it undefined?
what about "withStyles" and "withCaptions" params in IGetDataSourcesStatusParams interface?

Reply:
It is defined when the properties order has been configured and undefined otherwise.

Reply:
why not always defined with default to empty object?
why not keeping consistent with the other styles?

Reply:
It is simpler to keep it defined with a default empty object